File: Project1/Models/Rezerwacja.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Project1.Models
{
    public class Rezerwacja
    {
        public int id { get; set; }
        public int uzytkownik_id { get; set; }
        public int zasob_id { get; set; }
        public int rok { get; set; }
        public int mies { get; set; }
        public int dz { get; set; }
        public int godz { get; set; }
        public int min { get; set; }
    }
}

File: Project1/ClientApp/src/app/model/rezerwacja.ts
import { Injectable } from "@angular/core";

@Injectable()
export class Rezerwacja {
    public id: number;
    public uzytkownik_id: number;
    public zasob_id: number;
    public rok: number;
    public mies: number;
    public dz: number;
    public godz: number;
    public min: number;
}